### DISCUSSION TITLE
Got banned after 1 month of innactivity

### DISCUSSION THREAD

**Discussion by ApexScorpio** [YimMenu/YimMenu#2132] [General]
Hello, i dont play for a month, but yesterday i got an email saying my account was banned.

I didnt even used this menu too much. I used it 3 or 4 times to change my number of completed bunker resuplies (when they were glitched and didnt deliver the car). And mainly for protection, like kicking toxic players and cheaters. 
![rockstar lol](https://github.com/YimMenu/YimMenu/assets/145114589/88bbe255-3e1a-4b07-a752-f84877602ce3)

**Comment by lonelybud** [comment]
you can add the here 
https://github.com/YimMenu/YimMenu/issues/2124
following the template as others.
and may close the discussion
